init.lua

```diff
@@ -65,25 +65,17 @@ local function is_player(player)
 end
 
 local function set_player_attribute(player, key, value)
-	if player.get_meta then
-		local meta = player:get_meta()
-		if meta and value == nil then
-			meta:set_string(key, "")
-		elseif meta then
-			meta:set_string(key, tostring(value))
-		end
-	else
-		player:set_attribute(key, value)
+	local meta = player:get_meta()
```

Comment:
Here and in the function below: `get_meta` is guaranteed to return a valid object for valid players. Checks against `meta` should not be necessary.